test/powershell/engine/Job/Jobs.Tests.ps1

```diff
@@ -69,6 +77,18 @@ Describe 'Basic Job Tests' -Tags 'CI' {
             $ProgressMsg[0].StatusDescription | Should -BeExactly 2
         }
 
+        It 'Can use the user specified working directory parameter' {
+            $job = Start-Job -ScriptBlock { $pwd } -WorkingDirectory $TestDrive | Wait-Job
+            $jobOutput = Receive-Job $job
+            $jobOutput | Should -BeExactly $TestDrive.ToString()
+        }
+
+        It 'Throws an error when the working directory parameter is <case>' -TestCases $invalidPathTestCases {
+            param($path,$case)
+
+            {Start-Job -ScriptBlock { 1 + 1 } -WorkingDirectory $path} | Should -Throw
```

Comment:
```suggestion
            {Start-Job -ScriptBlock { 1 + 1 } -WorkingDirectory $path} | Should -Throw -ErrorId $expectedErrorId
```

Reply:
fixed in a8b2bfb8915b815a708a4b95711ea22b989e7159